.github/workflows/oscal-check.yaml

```
@@ -50,7 +50,7 @@ jobs:
           ./go-oscal validate --input-file oscal-sample.json 
 
       - name: Validate Catalog with oscal-club/NIST CLI
-        uses: oscal-club/oscal-cli-action@1a210b84bc1fd6adf15c9cf0d46a51d15a3d8301 # v2.0.1
+        uses: oscal-club/oscal-cli-action@b5b0c80a1a158797bea4475d13d12c494b56019b # v2.0.2
```

Just a heads up new action version dropped today, v2.1.0, which defaults to the newest CLI version from metaschema-framework fork.

Also those maintainers think there may be issues with the regex and filed bugs when they saw #450 and we chatted, but still green is green, you might want to get ahead of the dependabot bump in a day or two as you review this.

```suggestion
        uses: oscal-club/oscal-cli-action@6a8b6368885714e46fa1d4a65a6ce665902e08b7 # v2.1.0
```